pkg/service/one2onerestore/model_test.go

```diff
@@ -121,3 +121,60 @@ func TestValidateNodesMapping(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateKeyspaceFilter(t *testing.T) {
+	testCases := []struct {
+		name        string
+		keyspace    []string
+		expectedErr string
+	}{
+		{
+			name:     "Only keyspace level filtering",
+			keyspace: []string{"hello", "system"},
+		},
```

Comment:
We need to add a test for alternator tables (e.g. `Tab_le-With1.da_sh2-aNd.d33ot.-`).